src/timedata.cpp

```diff
@@ -105,11 +105,11 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
                 }
             }
         }
-        if (fDebug) {
-            BOOST_FOREACH(int64_t n, vSorted)
-                LogPrintf("%+d  ", n);
-            LogPrintf("|  ");
-        }
-        LogPrintf("nTimeOffset = %+d  (%+d minutes)\n", nTimeOffset, nTimeOffset/60);
+        
+        BOOST_FOREACH(int64_t n, vSorted)
+            LogPrint("net", "%+d  ", n);
+        LogPrint("net", "|  ");
```

Comment:
> Will this look weird when we decide to auto print the category?

Not if we stick with a strict format, and only print the category name on new lines along with the time.
Outputting something using multiple PrintX calls has always been possible, and should still be possible after that (currently purely theoretical) change.